metadata_annotation_gdsc.ipynb

```diff
@@ -0,0 +1,2420 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    drug_response = pt_metadata.drug_response_gdsc1[pt_metadata.drug_response_gdsc1['cell_line_name'].isin(adata.obs.cell_line)]
</pre><p>What's this mysterious <code>pt_metadata.drug_response_gdsc1</code> dataframe? Add a comment on it in the notebook please!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/pertpy-tutorials/pull/16/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>